Docs: Add clarifying note about disabled elements (#5520)
cypress-io/cypress-documentation: docs/guides/core-concepts/interacting-with-elements.mdx

### Disability

> Cypress checks whether an element's `disabled` property is `true`. Note that
> this property only affects
> [form controls](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled)
> such as `button` or `input`. Setting a `disabled` attribute on other elements will

Reviewer: ```suggestion
Cypress checks whether the `disabled` property is `true` on a
[form control](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled) element, such as `button` or `input`. Setting a `disabled` attribute on other elements will
```
